File: cookbook/typescript/automations/links/get.ts
import { config } from 'dotenv';
import { Retab } from '@retab/node';



async function main() {
  config({ path: '../../../.env.production' });
  
  const reclient = new Retab();
  
  const automation = await reclient.processors.automations.links.get({
    link_id: "lnk_Xf15nXFpo7mwfGT1aSYo4",
  });
  
  console.log(JSON.stringify(automation, null, 2));
  
}

main().catch(console.error);

File: cookbook/python/automations/mailboxes/get.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.mailboxes.get(
    mailbox_id="mb_FRf6FX5fYkenZ_JJlL5GD",
)

print(automation.model_dump_json(indent=2))
